manifests/2.3.0/opensearch-2.3.0.yml

```diff
@@ -25,3 +25,9 @@ components:
   - name: security
     repository: https://github.com/opensearch-project/security.git
     ref: 2.x
+  - name: performance-analyzer
+    repository: https://github.com/opensearch-project/performance-analyzer.git
+    ref: tags/2.3.0
```

Comment:
This shouldn't be 2.3 tag. This ref would be changed to use tag once the release launch. Right now it should point to the specific branch for 2.3.0.

Reply:
Ok. I will fix it and raise new PR.